packages/api/lambdas/sf-sns-broadcast.js

```diff
@@ -75,17 +74,6 @@ async function publish(message, finished = false) {
       } else {
         event.meta.status = 'completed';
       }
-      const granuleId = get(event, 'meta.granuleId', null);
-      if (granuleId) {
-        await setGranuleStatus(
-          granuleId,
-          event.meta.stack,
-          event.cumulus_meta.system_bucket,
-          event.cumulus_meta.state_machine,
-          event.cumulus_meta.execution_name,
-          event.meta.status
-        );
-      }
     } else {
       event.meta.status = 'running';
     }
```

Comment:
Yeah, but deprecating `sf-sns-broadcast` requires trying to refactor the `KinesisTriggerSpec` that depends on it. I'm totally fine to do it as part of this work, just didn't know if I should stop given the amount of changes already

Reply:
Right. That's fair. I don't think it's necessary to do it in this PR, or even under the scope of this work at all, since we have higher priorities. You may just want to file a ticket to clean it up.